client/modules/settings/components/General.js

```diff
@@ -56,7 +56,7 @@ const General = () =>
             rows="5"
           />
         </Col>
-        <Col lg={3}>
+        <Col lg={1}>
```

Comment:
Isn't it a bit small and weird looking having 1 and 2 as the width of these? Why not keep them all 3 and move the address to the end if it's in the way?